<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="printBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>展示些什么信息好呢</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>房间号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="roomBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>112</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="dayBtn">
      <property name="text">
       <string>日报表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="monthBtn">
      <property name="text">
       <string>周报表</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="yearBtn">
      <property name="text">
       <string>月报表</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="calWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>296</width>
     <height>236</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>851</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
